package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": false,
   "dependencies": {
-    "@data-driven-forms/pf4-component-mapper": "^0.4.2",
-    "@data-driven-forms/react-form-renderer": "^1.7.0",
+    "@data-driven-forms/pf4-component-mapper": "^0.4.1",
+    "@data-driven-forms/react-form-renderer": "^1.6.0",
```

Comment:
Did you have some issues with newer version of the renderer?

Reply:
No, I did not have any issues, updated to the new version